### PR TITLE
8 enable view password in the account settings

### PR DESCRIPTION
**Pull request by HildahN**
i used form->html and javascript

### CODE REVIEW

File: vendor/encore/laravel-admin/src/Controllers/AuthController.php [hunk 1]
```diff
@@ -126,18 +126,33 @@ public function putSetting()
...
+        $form->text('username', trans('admin.username'))->rules('required');
         $form->text('name', trans('admin.name'))->rules('required');
         $form->email('email', trans('admin.email'))->rules('required');
         $form->image('avatar', trans('admin.avatar'));
-        $form->password('password', trans('admin.password'))->rules('confirmed|required');
-        $form->password('password_confirmation', trans('admin.password_confirmation'))->rules('required')
-            ->default(function ($form) {
-                return $form->model()->password;
-            });
+        $form->html('<div class="input-group">
```
Comment (PriscillaCole) [inline vendor/encore/laravel-admin/src/Controllers/AuthController.php:138]: Did you test this, beacuse this auth controller is not the one used for the front end of the form for account settings

File: resources/js/app.js [hunk 2]
```diff
@@ -1 +1,15 @@
 import './bootstrap';
+        document.getElementsByClassName("eye-icon").addEventListener("click", function() {
```
Comment (PriscillaCole) [inline resources/js/app.js]: Whats this js for, if you wrote the entire script in the controller as well

Comment (HildahN) [reply resources/js/app.js:2]: its to enable the user to view their password when changing it

Comment (HildahN) [reply resources/js/app.js]: i must have forgotten to delete it.........i was first writing the js their so that it doesnt have errors

**Review by PriscillaCole — APPROVED** [review]

**Pull request merged**